src/test/java/uk/gov/pay/products/matchers/PaymentEntityMatcher.java

```diff
@@ -1,7 +1,7 @@
 package uk.gov.pay.products.matchers;
 
-import org.apache.commons.lang.ObjectUtils;
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.ObjectUtils;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Are we confident there’s no behaviour changes between the two versions for the methods we use?

Reply:
This matcher is exclusively used in tests, and all existing tests pass without any modification so I think it's safe to assume that the desired behaviour did not change. Also, lang3 release docs does not mention any behaviour changes.